libs/api/domains/form-system/src/models/applications.model.ts

```diff
@@ -0,0 +1,63 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql'
+import { Section } from './section.model'
+import { LanguageType } from './languageType.model'
+import { Dependency } from './form.model'
+
+@ObjectType('FormSystemApplicationEventDto')
+export class ApplicationEventDto {
+  @Field(() => Date, { nullable: true })
+  created?: Date
+
+  @Field(() => String, { nullable: true })
+  eventType?: string
+
+  @Field(() => Boolean, { nullable: true })
+  isFileEvent?: boolean
+}
+
+@ObjectType('FormSystemApplication')
+export class Application {
+  @Field(() => String, { nullable: true })
+  id?: string
```

Comment:
Are all of these fields actually nullable? When is a queried `application.id` null?